sass/slider.scss

```diff
@@ -110,6 +115,13 @@ $baseBorderRadius: 4px;
 	@include border-radius($baseBorderRadius);
 }
 
+.slider-track-left, .slider-track-right {
+  position: absolute;
+  background: transparent;
+  .box-sizing(border-box);

```

Comment:
Apparently not.  It should probably be

```@include box-sizing(border-box);```

I've never used SASS before and it's not in the default Grunt task, so I probably never ran it.